.github/workflows/build-container-image.yml

```diff
@@ -0,0 +1,139 @@
+name: Build & Publish Tobira image
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]          
+    tags:     [ 'v*' ]          
+
+permissions:
+  contents: read
+  packages: write               
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: elan-ev/tobira
+  DOCKERFILE: util/containers/Dockerfile
+
+jobs:
+  meta:
+    name: Compute Docker image tags & labels
+    runs-on: ubuntu-latest
+    outputs:
+      tags:   ${{ steps.meta.outputs.tags }}
+      labels: ${{ steps.meta.outputs.labels }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Docker metadata
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=sha,format=short,prefix=sha-
+            type=raw,value=edge,enable=${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}
+            type=semver,pattern={{version}},enable=${{ startsWith(github.ref, 'refs/tags/') }}
+            type=semver,pattern={{major}}.{{minor}},enable=${{ startsWith(github.ref, 'refs/tags/') }}
+            type=semver,pattern={{major}},enable=${{ startsWith(github.ref, 'refs/tags/') }}
+            type=raw,value=latest,enable=${{ startsWith(github.ref, 'refs/tags/') }}
+          labels: |
+            org.opencontainers.image.revision=${{ github.sha }}
+
+  build:
+    name: Build ${{ matrix.name }}
+    needs: meta
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: amd64
+            platform: linux/amd64
+            rust_target: x86_64-unknown-linux-musl
+          - name: arm64
+            platform: linux/arm64
+            rust_target: aarch64-unknown-linux-musl
```

Comment:
Why build for ARM? :thinking: Tobira never released for ARM, was never tested for ARM and is very unlikely to be used on an ARM machine.